x-pack/plugins/security_solution/public/timelines/components/side_panel/index.tsx

```diff
@@ -74,11 +74,13 @@ export const DetailsPanel = React.memo(
 
     let visiblePanel = null; // store in variable to make return statement more readable
     let panelSize: EuiFlyoutProps['size'] = 's';
+    let flyoutUniqueKey: string = timelineId;
```

Comment:
```suggestion
    let flyoutUniqueKey = timelineId;
```
💅 TypeScript should determine this type automatically